File: 0003_ArithmeticOperations/Java/AOpr.java
/* Problem Statement: Program for Arithmetic Operations using various methods */
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class AOpr 
{
	static int add(int x, int y)
	{
		int add = x + y;
		return add;
	}
	
	static int sub(int x, int y)
	{
		int sub = x - y;
		return sub;
	}
	
	static int mult(int x, int y)
	{
		int mult = x * y;
		return mult;
	}
	
	static int div(int x, int y)
	{
		int div = x / y;
		return div;
	}
	
	public static void main(String []args) throws NumberFormatException, IOException
	{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		String ans;
		int choice, a, b, result;
		System.out.println("Arithmetic Operations on Two Numbers");
		do
		{
			System.out.println("Press 1: Addition");
			System.out.println("Press 2: Subtraction");
			System.out.println("Press 3: Multiplication");
			System.out.println("Press 4: Division");
			System.out.println("Press 5: Quit");
			System.out.print(": ");
			choice = Integer.parseInt(br.readLine());
			
			switch(choice)
			{
			case 1:
				System.out.print("Enter First Number: ");
				a = Integer.parseInt(br.readLine());
				System.out.print("Enter Second Number: ");
				b = Integer.parseInt(br.readLine());
				result = add(a, b);
				System.out.println("Addition of " + a + " and "+ b +" is: "+result);
				break;
				
			case 2:
				System.out.print("Enter First Number: ");
				a = Integer.parseInt(br.readLine());
				System.out.print("Enter Second Number: ");
				b = Integer.parseInt(br.readLine());
				result = sub(a, b);
				System.out.println("Subtraction of " + a + " and "+ b +" is: "+result);
				break;
				
			case 3:
				System.out.print("Enter Multiplicand: ");
				a = Integer.parseInt(br.readLine());
				System.out.print("Enter Multiplier: ");
				b = Integer.parseInt(br.readLine());
				result = mult(a, b);
				System.out.println("Multiplication of " + a + " and "+ b +" is: "+result);
				break;
				
			case 4:
				System.out.print("Enter Dividend: ");
				a = Integer.parseInt(br.readLine());
				System.out.print("Enter Divisor: ");
				b = Integer.parseInt(br.readLine());
				result = div(a, b);
				System.out.println("Division of " + a + " and "+ b +" is: "+result);
				break;
				
			case 5:
				System.exit(0);
				break;
				
			default:
				System.out.println("Wrong choice entered!! PLease try again..");
				break;
			}
			
			System.out.println("Do you want to continue?");
			System.out.print("(Y/N): ");
			ans = br.readLine();
			
		}
		while(ans.toLowerCase().equals("y"));
	}
	
}

File: 0003_ArithmeticOperations/CPP/AOpr.cpp
/* Problem Statement: Program for Arithmetic Operations using various methods */
#include<iostream>

using namespace std;

class AOpr
{
public:
	int add(int x, int y);
	int sub(int x, int y);
	int mult(int x, int y);
	int div(int x, int y);
};

int AOpr :: add(int x, int y)
{
	int add = x + y;
	return add;
}
	
int AOpr :: sub(int x, int y)
{
	int sub = x - y;
	return sub;
}
	
int AOpr :: mult(int x, int y)
{
	int mult = x * y;
	return mult;
}
	
int AOpr :: div(int x, int y)
{
	int div = x / y;
	return div;
}
	
int main()
{
	AOpr obj;
	char ans;
	int choice, a, b, result;
	cout << "Arithmetic Operations on Two Numbers" <<endl;
	do
	{
		cout << "Press 1: Addition" <<endl;;
		cout << "Press 2: Subtraction" <<endl;
		cout << "Press 3: Multiplication" <<endl;
		cout << "Press 4: Division" <<endl;
		cout << "Press 5: Quit" <<endl;
		cout << ": ";
		cin >> choice;
			
		switch(choice)
		{
		case 1:
			cout << "Enter First Number: ";
			cin >> a;
			cout << "Enter Second Number: ";
			cin >> b;
			result = obj.add(a, b);
			cout << "Addition of " << a << " and " << b << " is: " << result << endl;
			break;

		case 2:
			cout << "Enter First Number: ";
			cin >> a;
			cout << "Enter Second Number: ";
			cin >> b;
			result = obj.sub(a, b);
			cout << "Subtraction of " << a << " and " << b << " is: " << result << endl;
			break;
				
		case 3:
			cout << "Enter Multiplicand: ";
			cin >> a;
			cout << "Enter Multiplier: ";
			cin >> b;
			result = obj.mult(a, b);
			cout << "Multiplication of " << a << " and " << b << " is: " <<result <<endl;
			break;
				
		case 4:
			cout << "Enter Dividend: ";
			cin >> a;
			cout << "Enter Divisor: ";
			cin >> b;
			result = obj.div(a, b);
			cout << "Division of " << a << " and " << b << " is: " <<result <<endl;
			break;
				
		case 5:
			return 0;
			break;
				
		default:
			cout << "Wrong choice entered!! PLease try again.." <<endl;
			break;
		}
			
		cout << "Do you want to continue?" <<endl;
		cout << "(Y/N): ";
		cin >> ans;
	}
	while(ans == 'y' || ans == 'Y');
}
